docs/coreConcepts/LitActionsAndPKPs/intro.md

```diff
@@ -12,9 +24,9 @@ We're building this at Lit: The smart contracts are Lit Actions and the key pair
 
 ## Programmable Key Pairs (PKPs)
 
-PKPs are public/private key pairs generated by the Lit network in a process called Distributed Key Generation. Each node custodies a *share* of the underlying private key, meaning the key **never exists in its entirety**. Currently, users can mint a PKP in the form of an ERC-721 NFT. The owner of the NFT becomes the sole controller of the underlying private key, their wallet signature becoming the method of authorization for the Lit nodes.
+PKPs are public/private key pairs generated by the Lit network in a process called Distributed Key Generation. Each node custodies a *share* of the underlying private key, meaning the key **never exists in its entirety**. Currently, users can mint a PKP in the form of an ERC-721 NFT. Only those with authorized access to sign with the PKP (via an [auth sig](/SDK/Explanation/WalletSigs/sessionSigs)) can combine the underlying key shares.
```

Comment:
For simplicity, I'd remove the "via an authSig" since that's not as simple as authSig! There are probably 2-3 steps to authorize someone who doesn't own the PKP to sign with it.